source/js/form.js

```diff
@@ -0,0 +1,43 @@
+(function() {
+  window.$ = $;
+  const email = $('#email')[0];
+  const city = $('#city')[0];
+  const name = $('#name')[0];
+  const formMap = {name, city, email};
+  Object.keys(formMap).map(key => !formMap[key] && delete formMap[key]);
+
+  const buttonSend = $('#send-form')[0];
+  buttonSend.onclick = e => {
+    e.preventDefault();
+    const validationFail = validation();
+    if (!Object.keys(validationFail).length) {
+      $.post(`/cities/${name}`, $('#form').serialize())
```

Comment:
`POST /cities` does not use the request paramenter. In order to create a city the request body is used;
```
const city = await m.city.create(req.body);
```

And where is the user creation being handled?